tests/org.aposin.mergeprocessor.test/src/org/aposin/mergeprocessor/configuration/JUnitConfiguration.java

```diff
@@ -41,7 +41,14 @@
     private Column sortColumn = Column.COLUMN_HOST;
     private int sortDirection = 1 << 10; //see SWT.DOWN
 
-    /**
+    private String renameDatabaseUrl = "jdbc:h2:C:\\dev\\eclipseworkspaces\\mergeprocessor\\repo\\tests\\org.aposin.mergeprocessor.test\\testH2\\test";
+
+	private Path localH2RenameDatabase = null;
```

Comment:
Also shifted - please fix before merging.

Reply:
Fixed and squashed.